signalfx-java/src/main/java/com/signalfx/signalflow/StreamRequestException.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2016 SignalFx, Inc. All rights reserved.
```

Comment:
Not familiar with an official policy, but in the past I've been instructed to bump copyright years on modification/creation.